README.md

```diff
@@ -1,23 +1,52 @@
 
-Everyone is welcome to participate, and no judgement will be made on members of the greater Mac community who are not a part of the community.
+-MacAdmins Slack Administrative Team 
 
-Be welcoming of everyone: discrimination based on gender identity, race, religion, nationality, political beliefs, sexual preference, preferred technological ecosystem, and concerns that have not been included in this list will be addressed with progressive discipline based on the severity of the infraction as determined by the administrative team.
+---
 
-Harassment, hate speech, bullying, sea-lioning, and gas-lighting are prohibited activities and are grounds for suspension from the community when there is clear documentation in public channels. If you experience any of these behaviors in direct messages please immediately contact the administrative team.
+## Code of Conduct
+All are welcome to participate within the Slack instance as long as the tenets of this Code of Conduct are adhered to.
 
-Your direct messages are only viewable by yourself and the other involved party and no one else. That being said, this is a community for professional communication and should be treated accordingly, regardless of the number of animated GIFs.
+### Policies
+The MacAdmins Slack Team is dedicated to providing a harassment-free experience for everyone, regardless of (but not limited to): gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or preferred technological ecosystem. 
 
-If you have concerns regarding this Slack instance please feel free to contact the administrative team and we will work to understand and address them respectfully and with an appropriate level of privacy to the given situation.
+We do not tolerate harassment of Slack participants in any form. Sexual language and imagery is not appropriate for any venue. Slack participants violating these rules may be sanctioned or expelled from the MacAdmins Slack at the discretion of the Administrative Team. 
 
-We hope that you find the community to be enriching to your personal and professional lives, and above all else a place of camaraderie where our collective knowledge and differences are celebrated in a positive, welcoming environment.
+Harassment includes, but is not limited to:
 
-Thank you for following this code of conduct. We reserve the right to amend or change the code of conduct at any time and encourage you to periodically review these guidelines to ensure a safe environment for all.
+* Verbal comments that reinforce social structures of domination related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or preferred technological ecosystem.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following 
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Inappropriate physical contact
```

Comment:
If there is ever an in-person event/meet-up related to the slack channel, be it informal or official then the CoC should still be in place as an expectation. Depending on your definition of "physical contact" this can also include expressed physical contact over the slack which would also be highly inappropriate.